app/views.py

```diff
@@ -3,12 +3,18 @@
 import re
 from json import loads
 
-from app import app, models, sql_session, viewsModel
```

Comment:
importing submodules rather than the classes and functions used within them can obscure the connections between modules. In this case, we see an enormous coupling between `views_model` and `views`; a large number of functions from the former are used across the entirety of the latter. 

First, I would say always import at the most specific level feasible; second, when you find yourself in this kind of situation, you can take it as an indication that you might want to restructure your modules to minimize coupling.